docker-compose-dev-linux.yml

```
@@ -16,7 +16,7 @@ services:
             - "6379:6379"
 
     eq-questionnaire-launcher:
-        image: onsdigital/eq-questionnaire-launcher:latest
+        image: onsdigital/eq-questionnaire-launcher:add-census-claims-to-ir-schema
```

Reminder to revert before merge
```suggestion
        image: onsdigital/eq-questionnaire-launcher:latest
```